### PR TITLE
feat: no-reserialize

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

**Comment by tpluscode** [comment]
I question the feature which automatically serialised in changed format TBH. Maybe we should revers the default behaviour in a breaking release?

File: packages/rdf-editor/src/RdfEditor.js [hunk 1]
```diff
@@ -179,7 +183,7 @@ export class RdfEditor extends Editor {
     })
 
     if (!quadStream) {
-      this.serialized = `No serializer found for media type ${this.format}`
+      this.value = `No serializer found for media type ${this.format}`
```
Comment (martinmaillard) [inline packages/rdf-editor/src/RdfEditor.js:186]: This has nothing to do with this feature right?

File: packages/rdf-editor/src/RdfEditor.js [hunk 2]
```diff
@@ -115,7 +119,7 @@ export class RdfEditor extends Editor {
       shouldSerialize = hasQuads
     }
 
-    if (shouldSerialize) {
+    if (shouldSerialize && !this.noReserialize) {
```
Comment (martinmaillard) [inline packages/rdf-editor/src/RdfEditor.js:122]: Does this only apply when changing format, or will it prevent any serialization when the content is updated?

File: packages/rdf-editor/src/RdfEditor.js (hunk 1 above)
Comment (tpluscode) [reply packages/rdf-editor/src/RdfEditor.js:186]: no. missed that in #45

File: packages/rdf-editor/src/RdfEditor.js (hunk 2 above)
Comment (tpluscode) [reply packages/rdf-editor/src/RdfEditor.js:122]: damn, I think you're right. must add a test to cover that bc it's apparently missing

**Review by martinmaillard — APPROVED** [review]

**Pull request merged**